presto-resource-group-managers/src/main/java/io/prestosql/plugin/resourcegroups/ResourceGroupSpec.java

```diff
@@ -70,14 +69,11 @@ public ResourceGroupSpec(
         this.name = requireNonNull(name, "name is null");
         checkArgument(maxQueued >= 0, "maxQueued is negative");
         this.maxQueued = maxQueued;
-        this.softConcurrencyLimit = softConcurrencyLimit;
-
-        checkArgument(hardConcurrencyLimit.isPresent() || maxRunning.isPresent(), "Missing required property: hardConcurrencyLimit");
-        this.hardConcurrencyLimit = hardConcurrencyLimit.orElseGet(maxRunning::get);
-        checkArgument(this.hardConcurrencyLimit >= 0, "hardConcurrencyLimit is negative");
-
+        checkArgument(hardConcurrencyLimit > 0, "hardConcurrencyLimit must be positive");
```

Comment:
While saying "positive" might be true from a mathematical perspective (I'm not sure), we should be more explicit

> must be greater than zero